### PR TITLE
Fix Symfony bundle load

### PR DESCRIPTION
**Pull request by jewome62**
Hello,

The bundle currently not working because : 

- ApiPlatformTranslationBundle.php is ouside of namespace scope from composer.json declaration
- The extension is bad named for autoloading by bundle

I change the alias, because that's remove error
``LogicException: Users will expect the alias of the default extension of a bundle to be the underscored version of the bundle name ("api_platform_translation"). You can override "Bundle::getContainerExtension()" if you want to use "api_translation" or another alias.
``

### CODE REVIEW

File: tests/ApiPlatformTranslationBundleTest.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Locastic\ApiTranslationBundle\Tests\Translation;
```
Comment (alanpoulain) [inline tests/ApiPlatformTranslationBundleTest.php]:
```suggestion
namespace Locastic\ApiTranslationBundle\Tests;
```

File: tests/ApiPlatformTranslationBundleTest.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+/**
+ * Class ApiPlatformTranslationBundleTest
+ *
+ * @package UnitTests\TranslationBundle
+ * @covers \Locastic\ApiPlatformTranslationBundle\ApiPlatformTranslationBundle
+ */
+class ApiPlatformTranslationBundleTest extends TestCase
+{
+    public function testClassExist()
```
Comment (alanpoulain) [inline tests/ApiPlatformTranslationBundleTest.php]:
```suggestion
    public function testClassExist(): void
```

File: tests/ApiPlatformTranslationBundleTest.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        $this->assertTrue(class_exists(ApiPlatformTranslationBundle::class));
+    }
+
+    public function testExtensionIsLoaded()
```
Comment (alanpoulain) [inline tests/ApiPlatformTranslationBundleTest.php]:
```suggestion
    public function testExtensionIsLoaded(): void
```

**Comment by antonioperic** [comment]
@jewome62 @alanpoulain thnx guys!

**Pull request merged**